### PR TITLE
[v7r2] Various fixes from 04.02.21 hachathon, mostly https related

### PR DESCRIPTION
**Pull request by chaen**
BEGINRELEASENOTES

Core
* CHANGE: improve tornado Logging
* FIX: fix credentials resolution with TornadoClient

DMS
* FIX: DFC base manager compatible with py3 string types and unicode

Resources
* FIX: explicitly cast gfal2 parameters to str (to be removed when moved to python 3)

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py [hunk 1]
```diff
@@ -509,15 +493,25 @@ def _request(self, retry=0, outputFile=None, **kwargs):
...
+        gLogger.error("No CAs found!")
+        return S_ERROR("No CAs found!")
       verify = self.__ca_location
 
     # getting certificate
+    # Do we use the server certificate ?
     if self.kwargs[self.KW_USE_CERTIFICATES]:
       cert = Locations.getHostCertificateAndKeyLocation()
+    # CHRIS 04.02.21
+    # TODO: add proxyLocation check ?
```
Comment (fstagni) [inline src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py:509]: Remove this comment?

File: src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py [hunk 2]
```diff
@@ -232,22 +231,7 @@ def __discoverCredentialsToUse(self):
 
     # ==== REWRITED FROM HERE ====
```
Comment (fstagni) [inline src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py:232]: "rewritten"

**Comment by fstagni** [comment]
... and a bit more explicit title please :)

File: src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py (hunk 1 above)
Comment (chaen) [reply src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py:509]: No no, I really have to do it, but first I need to review the real use case of that

**Review by atsareg — APPROVED** [review]

**Pull request merged**